src/c3chart/Combo.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["./CommonND"], factory);
+    } else {
+        root.c3chart_Combo = factory(root.c3chart_CommonND);
+    }
+}(this, function (CommonND) {
+    function Combo(target) {
+        CommonND.call(this);
+
+        this._type = "bar";
+        this._previousTypes = undefined;
+    }
+    Combo.prototype = Object.create(CommonND.prototype);
+    Combo.prototype.constructor = Combo;
+    Combo.prototype._class += " c3chart_Column";
+
+    Combo.prototype.publish("isStacked", false, "boolean", "Stack Chart",null,{tags:["Basic"]});
```

Comment:
@GordonSmith 

Notice that the **bar** values are stacked additively while the **line** values aren't effected by stack (because only one data set is using **line**):
http://rawgit.com/jbrundage/Visualization/C3-ComboChart/demos/dermatology.html?src/c3chart/Combo&types=bar,bar,line&isStacked=1